CalculateCalendarLogic/CalculateCalendarLogic.swift

```diff
@@ -159,9 +159,8 @@ public struct CalculateCalendarLogic {
      */
     public func judgeJapaneseHoliday(year: Int, month: Int, day: Int) -> Bool {
         
-        let cal = Calendar.current as NSCalendar
-        guard let date = cal.date(
-            era: AD, year: year, month: month, day: day, hour: 0, minute: 0, second: 0, nanosecond: 0) else {
+        let cal = Calendar(identifier: .gregorian)
+        guard let date = cal.date(from: DateComponents(era: AD, year: year, month: month, day: day, hour: 0, minute: 0, second: 0, nanosecond: 0)) else {
```

Comment:
__[SwiftLint]__

Line should be 120 characters or less: currently 153 characters


<!-- Comment by [Sider](https://sider.review) -->